src/util/runCommands.ts

```diff
@@ -0,0 +1,24 @@
+import { exec } from 'child_process';
+import { promisify } from 'util';
+
+const asyncExec = promisify(exec);
```

Comment:
Does this type of functionality not already exist somewhere int the code? If not then this is totally fine, just wanna make sure we don't have a bunch of code duplication.

Reply:
I have checked everywhere else looks like promisified exec is used, please lmk if you want me to change it